requirements/edx/base.txt

```diff
@@ -99,7 +99,7 @@ edx-django-release-util==0.4.4  # via -r requirements/edx/base.in
 edx-django-sites-extensions==2.5.1  # via -r requirements/edx/base.in
 edx-django-utils==3.7.3   # via -r requirements/edx/base.in, django-config-models, edx-drf-extensions, edx-enterprise, edx-rest-api-client, edx-when
 edx-drf-extensions==6.1.1  # via -r requirements/edx/base.in, edx-completion, edx-enterprise, edx-organizations, edx-proctoring, edx-rbac, edx-when, edxval
-edx-enterprise==3.6.4     # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in
+git+https://github.com/edx/edx-enterprise@8afb05a852762b92ba66567faea16d8b5e1a31c9#egg=edx-enterprise==3.6.3     # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in
```

Comment:
FYI, this is just for testing, will update the enterprise version after [PR In Enterprise](https://github.com/edx/edx-enterprise/pull/936) is merged and released.